README.md

```diff
@@ -67,9 +67,14 @@ config :sentry,
 | `release` | False  | None | |
 | `server_name` | False  | None | |
 | `use_error_logger` | False  | False | |
-| hackney_opts | False  | `[pool: :sentry_pool]` | |
-| hackney_pool_max_connections | False  | 50 | |
-| hackney_pool_timeout | False  | 5000 | |
+| `hackney_opts` | False  | `[pool: :sentry_pool]` | |
+| `hackney_pool_max_connections` | False  | 50 | |
+| `hackney_pool_timeout` | False  | 5000 | |
+| `enable_source_code_context` | True | | |
+| `root_source_code_path` | Required if `enable_source_code_context` is enabled | | Should generally be set to `File.cwd!`|
+| `context_lines` | False  | 3 | |
+| `exclude_patterns` | False  | `[~r"/_build/", ~r"/deps/", ~r"/priv/"]` | |
```

Comment:
good call.  to boot, this actually found a bug in that the option wasn't being used at all.